htdocs/projet/tasks/list.php

```diff
@@ -531,18 +531,28 @@
 if (!empty($arrayfields['t.dateo']['checked']))
 {
 	print '<td class="liste_titre center minwidth150">';
-	if (!empty($conf->global->MAIN_LIST_FILTER_ON_DAY)) print '<input class="flat" type="text" size="1" maxlength="2" name="search_sday" value="'.$search_sday.'">';
-	print '<input class="flat" type="text" size="1" maxlength="2" name="search_smonth" value="'.$search_smonth.'">';
-	$formother->select_year($search_syear ? $search_syear : -1, 'search_syear', 1, 20, 5);
+	print '<div class="nowrap">';
+	print $langs->trans('From').' ';
+	print $form->selectDate($search_dateo_start ? $search_dateo_start : -1, 'search_dateo_start', 0, 0, 1);
```

Comment:
Because the value ask a date to be used for sql filtering, the last parameter must be "tzuserrel"

Reply:
There is no such parameter for selectDate, is it ?

Reply:
Yes you have since v13. The parameter is called $gm
*  @param	mixed		$gm				'auto' (for backward compatibility, avoid this), 'gmt' or 'tzserver' or 'tzuserrel'

You must use tzuserrel